YoCodeAutomatedTests/BuildTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Xunit;
+using FluentAssertions;
+using YoCode;
+using System.IO;
+using System;
+
+namespace YoCodeAutomatedTests
+{
+    //Compares projects against junior-test unmodified project
+    public class BuildTests
+    {
+        [Theory]
+        [InlineData("\\P1.txt", "\\Project1")]
+        [InlineData("\\P2.txt", "\\Project2")]
+        [InlineData("\\P3.txt", "\\Project3")]
+        public void CompareProjects(string outputFile, string project)
+        {
+            var helper = new TestHelperMethods();
+
+            string argument = $"YoCode.dll --original={helper.testPath}\\TestProjects\\junior-test " +
+                $"--modified={helper.testPath}\\TestProjects{project}";
+
+            ProcessRunner pr = new ProcessRunner("dotnet", helper.dllPath, argument);
+            pr.ExecuteTheCheck("Minimum test count:");
+
+            var actualPath = helper.testPath+"\\ActualOutputs"+outputFile;
+            var expectedPath =helper.testPath+"\\ExpectedOutputs"+outputFile;
```

Comment:
You could save yourself a lot of escaped backslashes if you used `Path.Combine` to stick file paths together. It's also less error-prone.